dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateCopyTestJan2023.tsx

```diff
@@ -0,0 +1,153 @@
+import { cmp } from '@guardian/consent-management-platform';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodySeparator,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from './shared';
+
+export const SignInGateCopyTestJan2023 = ({
+	signInUrl,
+	guUrl,
+	dismissGate,
+	abTest,
+	ophanComponentId,
+	isMandatory = false,
+}: SignInGateProps) => {
+	const getCopyTextFromVariant: (abVariant: string | undefined) => string = (
+		abVariant,
+	) => {
+		switch (abVariant) {
+			case 'quick-and-easy': {
+				return "Register: it's quick and easy";
+			}
+			case 'take-a-moment': {
+				return 'Take a moment to register';
+			}
+			default: {
+				return 'You need to register to keep reading';
+			}
+		}
+	};
+	return (
+		<div css={signInGateContainer} data-cy="sign-in-gate-main">
+			<style>{hideElementsCss}</style>
+			<div css={firstParagraphOverlay} />
+			<h1 css={headingStyles}>
+				{getCopyTextFromVariant(abTest?.variant)}
+			</h1>
+			<p css={bodyBold}>
+				It’s still free to read - this is not a paywall
+			</p>
+			<p css={bodyText}>
+				We’re committed to keeping our quality reporting open. By
+				registering and providing us with insight into your preferences,
+				you’re helping us to engage with you more deeply, and that
+				allows us to keep our journalism free for all. You’ll always be
+				able to control your own{' '}
```

Comment:
It is a bit late… but am I dreaming or is this `privacy settings` link rendered using a fallback font (looking at the screenshots only)?

Reply:
Hmmm, I'm not sure to be honest. This component is based on `SignInGateMain` and this isn't something new being introduced with this PR. Though, happy to update if necessary?

Reply:
Necessary? No. But wrong (if true on site too and not only a local-only issue).

Reply:
It seems to be coming through as Arial.  Eagle eyes! We can raise a fix PR.